tbx/project_styleguide/templates/patterns/pages/wagtail/password_required.html

```diff
@@ -5,24 +5,26 @@
 {% block body_class %}template-password-required{% endblock %}
 
```

Comment:
To resolve the issue on dark mode, I think this will do the trick:

```suggestion
{% block theme_class %}{% endblock %}

```

Reply:
I've adjusted this slightly to go for `theme--light`